core/sdk-core/src/main/java/software/amazon/awssdk/core/client/handler/SdkAsyncClientHandler.java

```diff
@@ -20,6 +20,7 @@
 import software.amazon.awssdk.annotations.ThreadSafe;
 import software.amazon.awssdk.core.client.config.SdkClientConfiguration;
 import software.amazon.awssdk.core.client.config.SdkClientOptionValidation;
+import software.amazon.awssdk.core.internal.handler.BaseAsyncClientHandler;
```

Comment:
?

Reply:
I moved BaseAsyncClientHandler to internal namespace, that required an import where none was required before.